python/dolfin/fem/__init__.py

```diff
@@ -12,6 +12,9 @@
                                    SystemAssembler)
 from dolfin.fem.coordinatemapping import create_coordinate_map
 from dolfin.fem.dirichletbc import DirichletBC, AutoSubDomain
+from dolfin.fem.dofmap import (DofMap, make_ufc_dofmap,
```

Comment:
Do we need to import `make_ufc_foo`?

Reply:
I other `python/dolfin/` files I do `from dolfin import fem` and work directly with `fem.make_ufc_...`. I want to keep everything 2 level, even for internal wrapper calls.

Reply:
But feel free to discuss.

Reply:
Ok, let's export to 2nd level only public (be interfaced by users) classes. Done for dofmap.py